src/platform/tests/TestConfigurationMgr.cpp

```diff
@@ -55,15 +55,17 @@ static void TestPlatformMgr_Init(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
 }
 
-#if !defined(NDEBUG)
 static void TestPlatformMgr_RunUnitTest(nlTestSuite * inSuite, void * inContext)
 {
-    CHIP_ERROR err = CHIP_NO_ERROR;
+#if CHIP_DEVICE_LAYER_TARGET_OPEN_IOT_SDK
+    // TODO: Fix RunUnitTests() for Open IOT SDK.
```

Comment:
@ATmobica FYI, I don't know why enabling this test case was causing CI failures as the logs didn't contain any failed test cases, so I'm unable to investigate this myself.